Themes/default/index.template.php

```diff
@@ -212,22 +212,30 @@ function template_body_above()
 		if (!empty($context['user']['avatar']))
 			echo $context['user']['avatar']['image'];
 
-		echo $context['user']['name'], '</a>
+		echo '<span class="textmenu">', $context['user']['name'], '</span></a>
 					<div id="profile_menu" class="top_menu"></div>
 				</li>';
 
 		// Secondly, PMs if we're doing them
 		if ($context['allow_pm'])
 			echo '
 				<li>
-					<a href="', $scripturl, '?action=pm"', !empty($context['self_pm']) ? ' class="active"' : '', ' id="pm_menu_top">', $txt['pm_short'], !empty($context['user']['unread_messages']) ? ' <span class="amt">' . $context['user']['unread_messages'] . '</span>' : '', '</a>
+					<a href="', $scripturl, '?action=pm"', !empty($context['self_pm']) ? ' class="active"' : '', ' id="pm_menu_top">
+						<span class="main_icons inbox"></span>
+						<span class="textmenu">', $txt['pm_short'], '</span>', !empty($context['user']['unread_messages']) ? '
+						<span class="amt">' . $context['user']['unread_messages'] . '</span>' : '', '
+					</a>
 					<div id="pm_menu" class="top_menu scrollable"></div>
 				</li>';
 
 		// Thirdly, alerts
 		echo '
 				<li>
-					<a href="', $scripturl, '?action=profile;area=showalerts;u=', $context['user']['id'], '"', !empty($context['self_alerts']) ? ' class="active"' : '', ' id="alerts_menu_top">', $txt['alerts'], !empty($context['user']['alerts']) ? ' <span class="amt">' . $context['user']['alerts'] . '</span>' : '', '</a>
+					<a href="', $scripturl, '?action=profile;area=showalerts;u=', $context['user']['id'], '"', !empty($context['self_alerts']) ? ' class="active"' : '', ' id="alerts_menu_top">
+						<span class="main_icons alerts"></span>
```

Comment:
This icon is too ugly, I prefer _watched_ or _notify_button_
We could add it as _.main_icons.forum_alerts_ so it remains with some semantic context

Reply:
I'm fine with using a different icon, sure. But for the sake of maintaining consistency we should probably just change the icon for `.main_icons.alerts` in index.css, so that the same icon is used for alerts everywhere.

Reply:
Ehh I'll leave that up to you, doubt this icon is used in that many places, but it's certainly used for warnings too, which is odd

Reply:
It is odd, yes...

Reply:
What would really be good is a nice little bell icon. That's more or less standard on other platforms these days.

Reply:
We don’t have, I wouldn’t mind if we add the bell icon or sprite separately, the set has a bunch of bell icons (with the actions too)

Reply:
Done.